src/serverExplorer.ts

```diff
@@ -346,7 +346,9 @@ export class ServerExplorer implements TreeDataProvider< Protocol.ServerState |
             const state: Protocol.ServerState = item as Protocol.ServerState;
             const handle: Protocol.ServerHandle = state.server;
             const id1: string = handle.id;
-            const runState: string = this.runStateEnum.get(state.state);
+            const runState: string = (state.state === 2 && state.runMode === 'debug') ?
+                                    'Debugging' :
```

Comment:
I believe that we should have a test that asserts the fact that if the runMode is 'debug' the server label would show 'Debugging'. Changing this label in an inconspicuous move breaks the command visibility and no one would notice for a moment.